src/twig.core.js

```diff
@@ -913,15 +916,20 @@ var Twig = (function (Twig) {
      *
      */
     Twig.Templates.loadRemote = function(location, params, callback, error_callback) {
-        var id          = params.id,
-            method      = params.method,
-            async       = params.async,
-            precompiled = params.precompiled,
-            template    = null;
+        var id                  = params.id,
+            method              = params.method,
+            async               = params.async,
+            precompiled         = params.precompiled,
+            fetchTemplateSource = params.fetchTemplateSource,
```

Comment:
How about calling the option `format` or `parser`? We could introduce a function to add available parsers, e.g. to include markdown files.
```js
parser: "twig"
parser: "source"
parser: "markdown"
```

Just my ideas ...